flask_caching/__init__.py

```diff
@@ -733,7 +745,7 @@ def _memoize_kwargs_to_args(self, f, *args, **kwargs):
         new_args.extend(args[len(arg_names) :])
         return (
             tuple(new_args),
-            OrderedDict(
+            dict(
```

Comment:
Here `OrderedDict` was used to keep the dictionary in, well, ordered. Why do you think ordering is unnecessary here? Also, if it *is* unnecessary, the `sorted` operation of the next line is also not needed (and degrades performance).